### ISSUE TITLE
[application-layout] Accessibility issues

### ISSUE THREAD

**Issue by Blackop778** [cerner/terra-framework#229]
# Bug Report

## Description
The application layout component has started getting accessibility issues when tested with updated dependencies. The error is there are `<img>` tags without `alt` attributes.

It seems some dependency has updated which is triggering errors because this issue has just started occurring and happens on `master` branch.

## Steps to Reproduce
1. Change to `master` branch on git
2. NPM `install` or `clean:install` to update dependencies
3. Run webdriver tests on application layout

**Comment by Blackop778** [comment]
Closed in cerner/terra-core #1719
